project/Build.scala

```diff
@@ -115,7 +115,7 @@ object ApplicationBuild extends Build {
   //   libraryDependencies ++= provided(play.api, reactivemongo.driver)
   // )
 
-  lazy val common = project("common").settings(
+  lazy val common = project("common", Seq(chess)).settings(
```

Comment:
We could create a scalachess common module and depend on that. Let's save that for a follow up though.